src/assets/modifierdata/engineer.yaml

```
@@ -214,7 +225,7 @@
       text: Energy Amplifier
       modifiers:
         attributes:
-          Healing Power: [120, buff]
+          Healing Power: [250, buff]   #this is 250 instead of 120
```

```suggestion
          Healing Power: [250, buff]
```
Oh, yep, great catch!